### DISCUSSION TITLE
Automatic read more button J5

### DISCUSSION THREAD

**Discussion by rikaryoo** [joomla/joomla-cms#41189] [2. Cold Features]
Friends of Joomla, based on feedback from my clients who use Joomla as a website platform, they have reported a certain inconvenience with the read more button in the editor.

Template and extension companies have been working hard to organize text, but the truth is that the current default template for the Read More button has not evolved.

It would be interesting to think of a read more button with options for Joomla 5.

Suggestions would be as follows:

1 - Keep the button functions as they are today;
2 - Add an option to customize the amount of text characters, both for the title and for the body text within the category and in highlights;
3 - Option to put the title on the read more button or be able to edit this text in the button box.

I believe that expanding the customization options of the Read More button would greatly facilitate the development of a simple blog and the templates would solve the problems of when the text is without Read More in the first paragraph, the highlighted image goes down to the footer.

It's a suggestion that I believe is a simple one that would be very pleasing to beginner Joomla website developers.

I can collaborate with testing and translations, because I have limitations with programming.

**Comment by brianteeman** [comment]
This already exists in the article options

![image](https://github.com/joomla/joomla-cms/assets/1296369/dc36a61e-0ccc-4d01-a25a-9ecd92288f6b)

**Reply by rikaryoo** [reply]
That's right Brianteeman, but when I said customize the read more button, I was referring to something even more customized, within the editor a field could be added to put your own customization of the button text in each article without being the default added in that field that you showed.

And the plugin would have an extension for the body of the text of the category, the limitations there are only in the titles of the article.

Thanks for the feedback
